src/IsData.purs

```diff
@@ -0,0 +1,9 @@
+module IsData (class IsData) where
```

Comment:
This module should be re-exported from `Contract.PlutusData` since users might need to use it in a typeclass constraint somewhere